### PR TITLE
tools/ci: setup wasi-sdk/wamrc into cibuild

### PR DESCRIPTION
**Pull request by anchao**
## Summary

tools/ci: setup wasi-sdk/wamrc into cibuild

wasi-sdk:
https://github.com/WebAssembly/wasi-sdk
wamrc:
https://github.com/bytecodealliance/wasm-micro-runtime

Signed-off-by: chao an <anchao@xiaomi.com>

## Impact

N/A

## Testing

ci-check

### CODE REVIEW

File: tools/ci/testlist/sim-02.dat [hunk 1]
```diff
@@ -15,3 +15,6 @@
 
 # Do not build Windows configs
 -,sim:windows
+
+# Skip WebAssembly Micro Runtime
+-,sim:wamr
```
Comment (xiaoxiang781216) [inline tools/ci/testlist/sim-02.dat:20]: why remove from Ubuntu

Comment (anchao) [reply tools/ci/testlist/sim-02.dat]: Done, let us add wasi-sdk into cibuild

**Review by pkarashchenko — APPROVED** [review]

**Review by jerpelea — APPROVED** [review]

**Comment by yamt** [comment]
for what do you plan to use this?

**Comment by anchao** [comment]
> for what do you plan to use this?

Just run a small sandbox on NuttX to resolve some binary incompatibility issues through the neutral language such as WebAssembly

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by yamt** [comment]
> > for what do you plan to use this?
> 
> Just run a small sandbox on NuttX to resolve some binary incompatibility issues through the neutral language such as WebAssembly

i meant, why do you need these tools in the ci image?

**Comment by anchao** [comment]
> i meant, why do you need these tools in the ci image?
> 
> 

https://github.com/apache/nuttx-apps/pull/1609

**Pull request merged**